util/queuehelper/queue.go

```diff
@@ -0,0 +1,48 @@
+package queuehelper
```

Comment:
Usually having lots of "helpers" is an anti-pattern of the design. Is there a reason not to put it inside top level `queue.go` ?
 :pray: `queuehelper` -> `queue`, so `queuehelper.QueueJob` -> `queue.Job`

Reply:
Top level `queue.go`, in package `lookout`, created an import cycle.
I've moved it to `./queue/queue.go` in c6397e4.